x-pack/plugins/cases/public/components/connectors/servicenow/servicenow_sir_case_fields.tsx

```diff
@@ -29,250 +29,243 @@ const defaultFields: Fields = {
 
 const ServiceNowSIRFieldsComponent: React.FunctionComponent<
   ConnectorFieldsProps<ServiceNowSIRFieldsType>
-> = // TODO: Fix this manually. Issue #123375
-  // eslint-disable-next-line react/display-name
-  ({ isEdit = true, fields, connector, onChange }) => {
-    const init = useRef(true);
-    const {
-      category = null,
-      destIp = true,
-      malwareHash = true,
-      malwareUrl = true,
-      priority = null,
-      sourceIp = true,
-      subcategory = null,
-    } = fields ?? {};
+> = ({ isEdit = true, fields, connector, onChange }) => {
```

Comment:
note: use hide whitespaces changes option. Because of the removal of the `TODO` comment the whole component indent changed.